skywalking/trace/__init__.py

```diff
@@ -25,10 +25,13 @@
 
 
 class ID(object):
-    def __init__(self):
-        self.part1 = str(uuid.uuid1())
-        self.part2 = threading.get_ident()  # FIXME: thread ids are reused
-        self.part3 = int(time.time() * 1000) * 10000 + _id.next()
+    def __init__(self, string: str = None):
+        if string:
+            self.part1, self.part2, self.part3 = string.split('.')
```

Comment:
Today, we don't require the 3 parts as trace id anymore. No matter string or number. Trace id/segment id are only literal string.